_posts/research/2020-04-21-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Autonomous Coordination of Cooperative Multi-Robot Systems
+description: Reinforcement Learning for Distributed Control of Multiple Agents
+author: Joewie J. Koh
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 75%
+excerpt_separator: <!-- More -->
+---
+
+Robotic systems with heterogeneous teams of cooperating robots have the potential to fundamentally change the paradigm of robotization. 
```

Comment:
" change the paradigm of robotization ". 

1. You haven't defined any paradigm before
2. Robotization is too generic